olp-cpp-sdk-core/tests/client/OlpClientTest.cpp

```diff
@@ -39,7 +39,7 @@ using ::testing::_;
 class CallApiWrapper {
  public:
   virtual HttpResponse CallApi(
-      std::string path, std::string method,
+      std::string path, olp::http::NetworkRequest::HttpVerb method,
```

Comment:
```suggestion
      std::string path, http::NetworkRequest::HttpVerb method,
```
Same for all others below.

Reply:
Not going to compile. The `olp` namespace is not used in the file scope